crates/prism-recipe/harness/tests/test_flops_attestation.py

```
@@ -0,0 +1,648 @@
+    assert all(0 <= i < 64 for i in a)
+    # Degenerate spans must not loop forever or crash.
+    assert F.secret_indices("s", 8, 1) == [0]
+    assert len(F.secret_indices("s", 100, 3)) == 3
+
+
+def test_probe_secret_resolution_order():
+    os.environ.pop("PRISM_FLOPS_PROBE_SECRET", None)
+    os.environ.pop("PRISM_EVAL_SECRET_SEED", None)
+    s, src = F.probe_secret("explicit-value")
+    assert (s, src) == ("explicit-value", "explicit")
+    os.environ["PRISM_EVAL_SECRET_SEED"] = "12345"
```

Ruff (0.16.1)

[error] Possible hardcoded password assigned to: "PRISM_EVAL_SECRET_SEED"

(S105)